src/platform/platformdependent.c

```diff
@@ -317,19 +321,28 @@ short sortScoreBuffer() {
     return mostRecentSortedLine;
 }
 
-// loads the BrogueHighScores.txt file into the scoreBuffer global variable
+void setHighScoresFilename(char *buffer) {
+    strncpy(buffer, gameConst->variantName, 100);
+    strncat(buffer, "HighScores.txt", 100);
```

Comment:
FYI, this code feels semantically wrong - we don't know anything about `*buffer` at this point. However, in this case it's hinted this is a filename buffer, so it's ok to be bounds-checked with `BROGUE_FILENAME_MAX`, IMO